src/client/v2/algod/models/types.ts

```diff
@@ -624,9 +637,9 @@ export class AccountParticipation extends BaseModel {
       typeof selectionParticipationKey === 'string'
         ? base64ToBytes(selectionParticipationKey)
         : selectionParticipationKey;
-    this.voteFirstValid = voteFirstValid;
-    this.voteKeyDilution = voteKeyDilution;
-    this.voteLastValid = voteLastValid;
+    this.voteFirstValid = ensureBigInt(voteFirstValid);
+    this.voteKeyDilution = ensureBigInt(voteKeyDilution);
+    this.voteLastValid = ensureBigInt(voteLastValid);
```

Comment:
Just so I understand how much scrutiny is required - all these changes were forced upon you by the compiler, right?  You didn't have to catch each required use of `ensureBigInt` by hand, correct?

Reply:
My bad, I meant to add https://github.com/algorand/js-algorand-sdk/pull/852#discussion_r1491320035 before requesting reviews -- this file is generated, not handwritten